<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reg_user</class>
 <widget class="QDialog" name="reg_user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>169</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="l_imie">
      <property name="text">
       <string>Imię: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="le_imie"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="l_nazwisko">
      <property name="text">
       <string>Nazwisko: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="le_nazwisko"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="l_mail">
      <property name="text">
       <string>E-mail:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="le_mail"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="l_login">
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="le_login"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="l_haslo">
      <property name="text">
       <string>Hasło:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="le_haslo">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="le_haslo_2">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="l_haslo_2">
      <property name="text">
       <string>Powtórz hasło:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>361</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="l_info">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_register">
      <property name="text">
       <string>Zarejestruj</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_back">
      <property name="text">
       <string>Wróc</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
